install.sh

```diff
@@ -51,52 +102,48 @@ function uninstall() {
         SUBCONFIGS="$(cd "$SERVICES_DIR/$SERVICE" && find . -mindepth 1 -type f)"
         for SUBCONFIG in $SUBCONFIGS ; do
             SUBCONFIG=$(sanitizePath "$SUBCONFIG")
-            echo "removing '/usr/lib/systemd/$SERVICE/$SUBCONFIG'"
-            (cd "/usr/lib/systemd/" && cd "$SERVICE" && rm -rf "$SUBCONFIG")
+            echo "removing '$DEST_DIR/lib/systemd/$SERVICE/$SUBCONFIG'"
+            rm -rf "$DEST_DIR/lib/systemd/$SERVICE/$SUBCONFIG" 2> "/dev/null" || true
         done
     done
 
-    rm "/usr/local/bin/fw-fanctrl"
-    ectool autofanctrl # restore default fan manager
-    rm "/usr/local/bin/ectool"
-    rm -rf "/etc/fw-fanctrl"
+    rm "$DEST_DIR/bin/fw-fanctrl" 2> "/dev/null" || true
+    ectool autofanctrl 2> "/dev/null" || true # restore default fan manager
+    if [ "$SHOULD_INSTALL_ECTOOL" = true ]; then
+        rm "$DEST_DIR/bin/ectool" 2> "/dev/null" || true
+    fi
+    rm -rf "$SYSCONF_DIR/fw-fanctrl" 2> "/dev/null" || true
+    rm -rf "/run/fw-fanctrl" 2> "/dev/null" || true
 
     uninstall_legacy
 }
 
-# move remaining legacy files
-function move_legacy() {
-    rm "/usr/local/bin/fanctrl.py" 2> "/dev/null" || true
-    (cp "$HOME/.config/fw-fanctrl"/* "/etc/fw-fanctrl/" && rm -rf "$HOME/.config/fw-fanctrl") 2> "/dev/null" || true
-}
-
 function install() {
-    cp "./bin/ectool" "/usr/local/bin"
-    cp "./fanctrl.py" "/usr/local/bin/fw-fanctrl"
-    chmod +x "/usr/local/bin/fw-fanctrl"
-    chown "$LOGNAME:$LOGNAME" "/usr/local/bin/fw-fanctrl"
-    mkdir -p "$HOME/.config/fw-fanctrl"
-    mkdir -p "/etc/fw-fanctrl"
+    uninstall_legacy
 
-    cp -n "./config.json" "/etc/fw-fanctrl" 2> "/dev/null" || true
+    mkdir -p "$DEST_DIR/bin"
+    if [ "$SHOULD_INSTALL_ECTOOL" = true ]; then
+        cp "./bin/ectool" "$DEST_DIR/bin/ectool"
+        chmod +x "$DEST_DIR/bin/ectool"
+    fi
+    mkdir -p "$SYSCONF_DIR/fw-fanctrl"
+    cp "./fanctrl.py" "$DEST_DIR/bin/fw-fanctrl"
+    chmod +x "$DEST_DIR/bin/fw-fanctrl"
 
-    move_legacy
+    cp -n "./config.json" "$SYSCONF_DIR/fw-fanctrl" 2> "/dev/null" || true
```

Comment:
I understand, but by doing this, it will be impossible to set the configuration path to `/etc/fw-fanctrl` as it will always depend on the dest directory which also changes other paths.

Reply:
Oh right, the DEST_DIR currently defaults to `/usr`, right? And we also have prefix default to `/usr`.

Can we default the prefix to `/usr/local` and the DEST_DIR to an empty value? And then make sure we use prefix for /lib and /bin consistently? This solves that issue and pulls another parallel to conventional build systems. What do you think about that?